src/components/Footer.js

```diff
@@ -44,33 +44,33 @@ const Footer = () => {
   return (
     <Grid className={muiGridBlockContainer} container justify="flex-end">
       <Grid item container xs={6} direction="column">
-        <img className={muiIconLogo} component="img" src={logo} alt="Ovio" />
+        <img className={muiIconLogo} src={logo} alt="Ovio" />
         <Grid>
-          {socialMedias.map(item => (
-            <a key={item.link} href={item.link} target="_blank" rel="noreferrer noopener">
-              <Icon component="img" className={lightIcon} src={item.image.src} alt={item.image.alt} />
+          {socialMedias.map(({ link, image: { imgSrc, imgAlt } }) => (
```

Comment:
No need to change this, but it's totally find to leave these as `{src, alt}` and then use `src={src}` and `alt={alt}` on the components.